dns/dnscrypt-proxy/src/opnsense/mvc/app/models/OPNsense/Dnscryptproxy/General.xml

```diff
@@ -136,5 +136,12 @@
         <serverlist type="CSVListField">
             <Required>N</Required>
         </serverlist>
+        <disabled_serverlist type="TextField">
+            <mask>/^([a-z0-9.\-]{1,70}$/</mask>
```

Comment:
US ASCII chars are used only, so no Unicode flag here.
No uppercase letters. Currently names can be up to 48 characters long, so I used `70` here to have some space.

Reply:
Missing closing comma here. Need to update